competition.js

```diff
@@ -52,6 +52,7 @@ router.use('/:competitionId', async (req, res, next) => {
 
 router.get('/:competitionId', async (req, res) => {
   var script = ''
+  var filename = ''
```

Comment:
Oops, as you can see my testing hasn't been thorough enough during my recent development work :)

Reply:
Just since this is fixing a breakage, I'm going to go ahead and merge this change in #48 while you're working on the change in lib.js.

Reply:
Thanks and sorry for the wait!